includes/utilities.inc

```diff
@@ -24,3 +24,54 @@ function presbydora_text_view(AbstractObject $object) {
     ),
   );
 }
+
+/**
+ * Function that will return an array of ancestors related to the pid param.
+ *
+ * @param string $pid
+ *   Object pid to find ancestors of.
+ *
+ * @return array
+ *   Array containing ancestors' PIDs.
+ */
+function presbydora_get_ancestors($pid) {
+  $qp = new IslandoraSolrQueryProcessor();
+  $query = format_string('PID:"!pid"',
+    array('!pid' => $pid)
+  );
+  $qp->buildQuery($query);
+  $qp->solrParams['fq'] = '';
+  $qp->executeQuery(FALSE);
+  try {
+    $results = $qp->islandoraSolrResult['response']['objects'];
+    $total_results = $qp->islandoraSolrResult['response']['numFound'];
+  }
+  catch (Exception $e) {
+    watchdog_exception('Presbydora', $e, 'Got an exception while searching for object ancestors and constituencies.', array(), WATCHDOG_ERROR);
+    $results = array();
+  }
+
+  $ancestors = array();
+  $constituencies = array();
+  if ($total_results > 0) {
```

Comment:
I'd add a comment as to the particular use case that could be encountered as to why we can't rely on `ancestors_ms` only, with an `XXX:` or some such.